lib/config.js

```diff
@@ -44,6 +44,49 @@ function putNestedValue(params, key, value) {
   return params;
 }
 
+function parseCloudinaryConfigFromEnvURL(ENV_STR) {
```

Comment:
Should we use camelcase naming convention for parameters and function names?

Reply:
Refactored to underscore all the public APIs, I kept internals as camelCase as there's a mixed style in the code already, might be worth slowly migrating to camelCase?